README.md

```diff
@@ -38,6 +38,19 @@ There are multiple channels other than `stable`. See the full list with
 for everyday charming. With the snap you will always be up to date as
 Charmhub services and APIs evolve.
 
+You also can install it from PyPI but some system packages and a specific 
```

Comment:
```suggestion
You also can install `charmcraft` from PyPI, but some system packages and a specific 
```

Reply:
I would also write... "You can also install..." ... but I am not a native English speaker.

Reply:
I would also remove `a specific` - I think it is clear, that you cannot install a random package :-)

Reply:
Thanks, all three improvements are great.